<ui version="4.0" >
 <class>ErrorListUi</class>
 <widget class="QDialog" name="ErrorListUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="4" >
    <widget class="QTextBrowser" name="descriptionTB" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>For more information, refer to the complete log.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="errorsLabel" >
     <property name="text" >
      <string>&amp;Errors:</string>
     </property>
     <property name="buddy" >
      <cstring>errorsLW</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QLabel" name="descriptionLabel" >
     <property name="text" >
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QPushButton" name="closePB" >
     <property name="text" >
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>121</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="viewLogPB" >
     <property name="toolTip" >
      <string>Open the LaTeX Log File dialog</string>
     </property>
     <property name="text" >
      <string>View Complete &amp;Log...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QPushButton" name="showAnywayPB" >
     <property name="toolTip" >
       <string>Attempt to show the output even if there were compilation errors</string>
     </property>
     <property name="text" >
      <string>Show Output &amp;Anyway</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4" >
    <widget class="QListWidget" name="errorsLW" >
     <property name="toolTip" >
      <string>Selecting an error will show the error message in the panel below and the cursor will jump to the location in the document where the error occurred.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>errorsLW</tabstop>
  <tabstop>descriptionTB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
